api/redhatopenshift/HcpCluster/hcpCluster-models.tsp

```diff
@@ -474,6 +493,19 @@ model NodePoolSpec {
   tuningConfigs?: string[];
 }
 
+/** Taint is controlling the node taint and its effects */
+model Taint {
+  /** The key of the taint */
+  @minLength(3)
+  key: label;
+
+  /** The value of the taint */
+  value?: label;
```

Comment:
I think saying that a taint value is a "label" and a taint key is a "label" can also be confusing.

K8s Taint key and values are a different concept than K8s Label key and values

Reply:
However I see that in CS we consider them as the same, including their validation

Reply:
I have done the changes to cover this.